packages/nvidia_nat_core/tests/nat/front_ends/fastapi/test_message_handler.py

```
@@ -87,6 +89,39 @@ async def test_context_manager_resolves_connection_identity_before_restoration()
     restore.assert_awaited_once()
 
 
+async def test_context_manager_uses_configured_identity_header():
+    """The configured identity header is passed through the verified connection resolver."""
+    handler, socket, _ = _make_message_handler(identity_header="X-User-ID")
```

Ruff (0.16.3)

[warning] Unpacked variable `socket` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)